### ISSUE TITLE
UnusedUses - invalid behaviour

### ISSUE THREAD

**Issue by f3l1x** [slevomat/coding-standard#60]
``` php
use ArrayAccess;
use Countable;
use Iterator;

class NodeList implements ArrayAccess, Countable, Iterator
{

    public function count()
    {
    }

    public function current()
    {
    }

    public function next()
    {
    }

    public function key()
    {
    }

    public function valid()
    {
    }

    public function rewind()
    {
    }

    public function offsetExists($offset)
    {
    }

    public function offsetGet($offset)
    {
    }

    public function offsetSet($offset, $value)
    {
    }

    public function offsetUnset($offset)
    {
    }

}
```

CodeSniffer result is:

```
 11 | ERROR | [x] Type Iterator is not used in this file
    |       |     (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
```

When I have changed order like this:

``` php
class NodeList implements Countable, Iterator, ArrayAccess
{
```

Result is:

```
 11 | ERROR | [x] Type ArrayAccess is not used in this file
    |       |     (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)
```

---

Can you help me please?

**Comment by TomasVotruba** [comment]
Could you send failing test please?

**Comment by f3l1x** [comment]
I'm not quite good at writing CodeSniffer tests, but I'll try to.

**Comment by TomasVotruba** [comment]
I'd just copy existing one and replace content:

- https://github.com/slevomat/coding-standard/blob/62d35e6049a60bbd85762852885aa7afec958e8e/tests/Sniffs/Namespaces/UnusedUsesSniffTest.php#L125-L131
- https://github.com/slevomat/coding-standard/blob/master/tests/Sniffs/Namespaces/data/unusedUsesAnnotationUnderscores.php

**Comment by f3l1x** [comment]
Is it already fixed by this https://github.com/slevomat/coding-standard/commit/a84c4ceec148ee42e53179f7ec253dddf33a98fe? :-) @pepakriz

**Comment by f3l1x** [comment]
I think it's fixed.

https://github.com/slevomat/coding-standard/pull/55

**Comment by f3l1x** [comment]
@ondrejmirtes I think problem still remains. If I use more then two implements.

```php
use ArrayAccess;
use Countable;
use IteratorAggregate;

class Foo implements ArrayAccess, Countable, IteratorAggregate
```

**Comment by kukulich** [comment]
It's fixed in branch 2.0-dev for sure.
